views/js/controller/DeliveryServer/awaiting.js

```diff
@@ -74,9 +74,23 @@ define([
 
             const runDelivery = () => {
                 loadingBar.start();
-                clipboard.clean();
-                deliveryStarted = true;
-                window.location.href = runDeliveryUrl;
+
+                // make sure that the delivery is still allowed (before redirect)
+                $.get(isAuthorizedUrl, (result) => {
+                    if (!result.success) {
+                        if (result.message) {
+                            dialogAlert(result.message, exit);
+                        } else {
+                            exit();
```

Comment:
from my perspective, exit without any notification is bad user experience. i think there still should be some generic notification about server error

Reply:
Fixed